config/webpack.config.js

```diff
@@ -80,7 +83,7 @@ var configGenerator = (options) => {
     },
     plugins: [
       new webpack.DefinePlugin({
-          __BUILDTYPE__: options.buildtype
+          __BUILDTYPE__: JSON.stringify(options.buildtype)
```

Comment:
Good catch!

Yes, that's been a long-standing bug.  It's actually set in travis now (wee .travis.yml) and should go away.

It didn't quite belong in the webpack.config.js because sometimes you do want to build the dev version.

Fingers crossed that it will work this time...